code/modules/antagonists/changeling/changeling_power.dm

```diff
@@ -32,6 +32,8 @@
 	var/ignores_fakedeath = FALSE
 	/// used by a few powers that toggle
 	var/active = FALSE
+	/// Does this ability stop working if you are burning?
+	var/disabled_by_fire = TRUE
```

Comment:
i'm wondering if this shouldn't be false by default... it might have a larger code fingerprint but i think having it be opt-in rather than opt-out makes more sense from a code-design standpoint... what are your thoughts?

Reply:
For me it makes more sense for it to be true as the amount of ling abilities disabled by it is more than the amount that arent so I dont have to have to set this var to true for most ling abilities but if I have to change it I can